great_expectations/rule_based_profiler/util.py

```diff
@@ -299,20 +343,37 @@ def compute_bootstrap_quantiles(
     bootstraps: np.ndarray = np.random.choice(
         metric_values, size=(n_resamples, metric_values.size)
     )
-    lower_quantile = np.mean(
-        np.quantile(
-            bootstraps,
-            q=false_positive_rate / 2,
-            axis=1,
-            interpolation="linear",  # can be omitted ("linear" is default)
-        )
+    lower_quantiles: Union[np.nd_array, Number] = np.quantile(
```

Comment:
@NathanFarmer What is `np.nd_array`?  I am aware of only `np.ndarray` -- could this be a typo?  Also, please note that we already have the types `MetricValue` (singular) and `MetricValues` (plural), which encompass the needed types.  I am also not sure whether or not we need multi-dimensional statistics right now -- we could use the same statistic computed for every dimension, though (is that what you meant)?  Happy to discuss.  Thank you.